tests/base_test.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.home import HomePage
+
+
+class BaseTest:
+
+    def login_new_user(self, mozwebqa, user):
```

Comment:
Regarding the asserts, we don't really need them for each test, but the issue is that we really should have them for at least one test. The actual test for logging in, `test_that_a_new_user_can_login` now calls this method to do the log in, so if we remove the asserts from here we will lose them from that test as well, unless we duplicate these steps in that test, which seems redundant. I could just remove the asserts from here and rely on an error happening if there is a problem logging in, but I don't really like that either.

To me the most efficient thing, in terms of code maintenance, is to keep this as is, even though it means each test will end up calling a few extra assertions. If you have a suggestion that addresses this issues I have raised here I'm happy to hear it.